app/src/main/res/resource/values-de/strings.xml

```diff
@@ -223,21 +225,21 @@
     <string name="action_align_right">Rechts anheften</string>
```

Comment:
https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR212
"einreichen"? Maybe "übermitteln" is better as this is not that formal?